UpskillWikipediaUITests/XCUITestHelpers/BaseTestCase.swift

```diff
@@ -0,0 +1,30 @@
+import XCTest
+
+class BaseTestCase: XCTestCase {
+
+    var app: XCUIApplication!
+    private lazy var skipButton = app.staticTexts["Skip"]
+    
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        setupRegionAndLocaleToUK()
+        app.launch()
+        skipOnboarding()
+    }
+    
+    func setupRegionAndLocaleToUK() {
+            app.launchArguments = ["-AppleLanguages", "(en)"]
+    }
+    
+    func skipOnboarding() {
+        if skipButton.exists {
+            skipButton.tap()
+        }
+    }
+    
+    func dumpViewHierarchy() {
```

Comment:
Use this function in `tearDown` method to capture the view hierarchy in case of test failure.